### PR TITLE
Implement UI for Currency Picker form

### PR DESCRIPTION
**Pull request by rdelrosariojr**
## Related Links:
- Issue link

## Definition of Done
- [ ] Follow UI from Currency picker form currency_picker package

## Notes:
- installed new package
- This will not trigger a change of currency yet, it needs backend

## Screenshots
image
image

## Test View  Points
- [ ] Currency code in the field will change if you select other currency

### CODE REVIEW

File: lib/consts/global_style.dart [hunk 1]
```diff
@@ -6,7 +6,7 @@ class AppColor {
   static const Color secondary = Color(0xFF324ED9);
   static const Color lightGray = Color(0xFFA7ABB8);
   static const Color gray = Color(0xFF666666);
-  static const Color darkBlue = Color(0xFF3D3541);
+  static const Color darkBlue = Color(0xFF242D58);
```
Comment (AubreyV) [inline lib/consts/global_style.dart]: This has same value as `onyx`. Maybe we can use `onyx` na lang for this value.

Comment (rdelrosariojr) [reply lib/consts/global_style.dart]: @AubreyV Actually they were interchanged ma'am, onyx is 3D3541 and dark blue is 242D58

Comment (AubreyV) [reply lib/consts/global_style.dart]: @rdelrosariojr oki pls apply this. si onyx dapat `3D3541` and dark blue is `242D58`

Comment (louisepujante) [reply lib/consts/global_style.dart]: @rdelrosariojr this has been fixed in the [[UI] Implemented tab layout with dynamic content #16 PR](https://github.com/framgia/sph_flutter-capstone/pull/16/files#diff-4451a9249b8acf87752363610d509d955736eccb1b9550071b34c360b1bb6b68)

File: lib/consts/global_style.dart [hunk 2]
```diff
@@ -8,6 +8,7 @@ class AppColor {
   static const Color gray = Color(0xFF666666);
   static const Color onyx = Color(0xFF3D3541);
   static const Color darkBlue = Color(0xFF242D58);
+  static const Color onyx = Color(0xFF3D3541);
```
Comment (AubreyV) [inline lib/consts/global_style.dart]: remove na ni coz naa na si onyx sa line 9

Comment (rdelrosariojr) [reply lib/consts/global_style.dart]: @AubreyV haiii

**Review by AubreyV — APPROVED** [review]

**Review by louisepujante — APPROVED** [review]

**Pull request merged**